src/Sequence.ts

```diff
@@ -526,6 +528,35 @@ export class Sequence<TItem> implements Iterable<TItem> {
     return new Sequence(createTakeWhileIterable(this._iterable, predicate));
   }
 
+  /**
+   * Returns elements from a sequence as long as they don't exist in the specified iterable items.
+   *
+   * @param items     The provided set of items that should not be in the returned Sequence.
+   * @param predicate The optional predicate that determines if two TItem items are equal.
+   * @returns Sequence<TItem>
+   *
+   * @example
+   * ```ts
+   * // returns [2, 4, 6]
+   * from([1, 2, 3, 4, 5, 6])
+   *   .without([1, 3, 5])
+   *   .toArray();
+   *
+   * // returns [1, 3, 5]
+   * from([1, 2, 3, 4, 5, 6])
+   *   .without([1, 3, 5], (item, withoutItem) => item === (withoutItem + 1))
+   *   .toArray();
```

Comment:
Took me a little while to understand this second example. Maybe something like this could be more understandable. What do you think?

```ts
// returns [{ id: 1 }, { id: 3 }]
from([{ id: 1 }, { id: 2 }, { id: 3 }])
  .without([{ id: 2 }], (a, b) => a.id === b.id)
  .toArray();
```